frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -114,6 +147,8 @@ const Input = (props: TaipyInputProps) => {
                         val = max;
                     }
                     setValue(val.toString());
+                    updateValueWithDelay(val);
+                    onAction && dispatch(createSendActionNameAction(id, module, onAction, evt.key, updateVarName, val));
```

Comment:
And I don't think we want to call the onAction when the user presses Arrow up or down ?